### DISCUSSION TITLE
Dismiss Sticky Overlay ID Help

### DISCUSSION THREAD

**Discussion by ghost** [adamk33n3r/runelite-watchdog#130] [Q&A] [answered]
I am trying to set up a simple notification using an overlay with the sticky option. The goal is to trigger the overlay when I am out of combat and dismiss it once I aggro monsters again. The sticky option is perfect for my situation because of the time tracking it provides. However I am confused on how to set this up correctly because once I toggle on sticky, it requests an ID to use with the dismiss overlay notification. I've tried playing around with these two notification settings but just unable to figure to out. Any clarity on what ID is needed and possibly an example on how to dismiss the sticky overlay would be really helpful! Thanks

**Comment by adamk33n3r** [comment, marked as answer]
The id on the Overlay notification that pops up when you turn sticky on can be anything you want. It just needs to match the one you put in the dismiss one. So you'll just have 2 alerts, one with Overlay, and one with Dismiss Overlay. Both id's matching.
